models/ProfileModel.js

```diff
@@ -1,11 +1,21 @@
-const getUserPosts = `WITH post_likes AS (
-    SELECT "postId", 
+const editProfile = `
+  UPDATE app_user
+  SET "displayName" = $1,"birthDate" = $2, "bio" = $3
+  WHERE "userId" = $4
+  RETURNING "displayName", "birthDate", bio;
```

Comment:
Query seems concise enough, I think `bio` at the `RETURNING` statement needs to be `"bio"`

Reply:
I actually ended up removing this because we can just use the data from the input fields to update the client side